drop-in/src/main/java/com/adyen/checkout/dropin/ui/viewmodel/DropInViewModel.kt

```diff
@@ -121,6 +127,9 @@ internal class DropInViewModel(
             savedStateHandle[CURRENT_ORDER] = value
         }
 
+    private val orderStatusRepository: OrderStatusRepository
```

Comment:
Why not put these in the constructor?

Reply:
The default value should indeed be removed. But all constructor arguments can be provided from `DropInViewModelFactory`. Everything is already setup correctly to make that work

Reply:
The default value is needed because we were getting a crash without it, since there is no zero argument constructor anymore. I will try to reproduce.